tools/run-tests.py

```diff
@@ -210,12 +206,14 @@ def report_command(cmd_type, cmd, env=None):
     sys.stderr.write(f"{TERM_BLUE}" +
                      f" \\{TERM_NORMAL}\n\t{TERM_BLUE}".join(cmd) +
                      f"{TERM_NORMAL}\n")
+    sys.stderr.flush()
 
 def report_skip(job):
     sys.stderr.write(f'{TERM_YELLOW}Skipping: {job.name}')
     if job.skip:
         sys.stderr.write(f' ({job.skip})')
     sys.stderr.write(f'{TERM_NORMAL}\n')
+    sys.stderr.flush()
```

Comment:
Are these needed for windows?

Reply:
It's needed for the CI can flush the output immediately, otherwise, we can not see any error message before the CI finished. That's the issue of windows msvcrt cached the output when use PIPE, when python subprocess using it, we are using PIPE to capture stdout stderr